Docs/source/pages/commands.md

```diff
@@ -1353,6 +1565,24 @@ Simulates a swipe action between two points.
                 By.NAME, "Drop").get_component_property("UnityEngine.UI.Image", "sprite", "UnityEngine.UI")
             assert image_source != image_source_drop_zone
 
+    .. code-tab:: robot
+
+        Test Resize Panel
+            ${alt_object}=    Find Object    NAME    Resize Zone
+            ${alt_object_x}=    Get Object X    ${alt_object}
```

Comment:
I would use the same example as other languages.

Reply:
Used corresponded test example from robot library.